modules/apps/object/object-test/src/testFunctional/tests/ObjectDisplayPageTemplates.testcase

```diff
@@ -43,4 +43,31 @@ definition {
 			value1 = "Custom Object 205");
 	}
 
+	@description = "LPS-137871 - Verify it is not possible to select an unpublished Object for a Page Template"
+	@priority = 4
+	test Totoya {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 290 Test",
+			objectName = "CustomObject290",
+			pluralLabelName = "Custom Objects 290");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldBusinessType = "Text",
+			fieldLabelName = "Custom Field",
+			fieldName = "customObjectField",
+			fieldType = "String",
+			isRequired = "false",
+			objectName = "CustomObject290");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject290");
+
+		DisplayPageTemplatesAdmin.openDisplayPagesAdmin(siteURLKey = "guest");
+
+		LexiconEntry.gotoAdd();
+
+		LexiconCard.clickCard(card = "Blank");
+
+		Click(locator1 = "VisualizeObjectDisplayPage#OBJECT_CONTENT_TYPE");
+	}
+
```

Comment:
Can you remove this code from the buildService commit? This type of commit can only contain generated code.